dockerfiles/Dockerfile.use-installer

```diff
@@ -0,0 +1,76 @@
+ARG IMAGE="python:3-slim-buster"
+
+#---
+# Place anything that is common to both the build and execution environment in base 
+#
+FROM $IMAGE AS base
+
+RUN apt-get update -qq \
+ && DEBIAN_FRONTEND=noninteractive apt-get -y install --no-install-recommends \
+    ca-certificates \
+    git \
+    libusb-1.0-0 \
+    make \
+    wget \
+ && apt-get autoclean && apt-get clean && apt-get -y autoremove \
+ && update-ca-certificates \
+ && rm -rf /var/lib/apt/lists
+
+# FROM base as builder 
+# Install necessary tools and source code and build from source here for ci/cd
```

Comment:
Overall, I feel there are more comments than content in the dockerfile. I believe this should not be an introduction to Docker, containers or dockerfiles. Therefore, I suggest moving the commented dockerfile to a markdown document, and preserving necessary comments only.